### ISSUE TITLE
[suggestion] clarification of java branch

### ISSUE THREAD

**Issue by Dr-Flay** [leolewis/openvisualtraceroute#22]
The site and docs need more clarity
Saying "at least java 11" isn't as useful as it should be with other software.

Java versioning is a confusing landscape for the average user.
Most users are not developers so install the regular runtime files from https://www.java.com/en/download/manual.jsp
This is version 8 branch, so Visual Trace Route fails to run, shows an error and sends the user to the oracle site for a developer version of java.
Here we are presented with a myriad of options depending on where on the page you look.
Some may be free but the language it is not as clear as it could be.
The offered version are 17 and 20 with a perplexing end of life being reversed as the v17 will be updated longer than 20.
Once scrolling down the page we finally find version 11 and wonder if and why that lower number has a longer shelf life than both 17 and 20.

So in short some explanation to an average end-user, of the requirement of which version between SE, JDK, JRE is needed
This software will not work with the common regular newest end user java, but instead requires a specific developer branch.

**Comment by leolewis** [comment]
Thanks for the feedback, that is a very good point you're making and has been the main point of friction for users.
At the time the software was first written, there was only one main version of java and things were simpler to setup, but the java landscape certainly got more complicated in the last few years (especially across the different OSs).
One option I was evaluating was to embed a JRE in the application, which is getting more and more common for JVM based application, and would remove that friction, but I am still debating on the possible side effects of having to maintain that runtime in case a critical bug is found.
If I find a reasonable solution, I will try and integrate it in the next release.

**Comment by Dr-Flay** [comment]
I'm glad you take my point in the spirit it was intended, as I don't see you as being at fault.
I feel for you, being caught between a rock and a hard place
SDKs are supposed to make things easier.

**Comment by leolewis** [comment]
Sorry for the delay, I just packaged [build 2.1.0 for windows](https://sourceforge.net/projects/openvisualtrace/files/2.0.0/OpenVisualTraceroute2.1.0.exe/download), which has the right JRE embedded.
That should solve that issue for Windows users, which is 80% of the users base, and will work on applying that paradigm to the Mac version and the various linux versions when I have some bandwidth.
